WBooks/Screens/BookLibrary/BookLibraryViewModel.swift

```diff
@@ -10,28 +10,49 @@ import UIKit
 
 class BookLibraryViewModel {
     
-    var libraryItems: [Book] = [] {
+    var bookLibraryModel: [Book] = [] {
         didSet {
-            self.reloadViewClosure?()
+            reloadViewClosure?()
         }
     }
     
-    var numberOfCells: Int {
-        return libraryItems.count
+    private var bookDetailsModel: [BookDetails] = [] {
```

Comment:
Whats the difference between `bookDetailsModel` and `bookLibraryModel`  ?  If i'm ok, `BookDetails` as a `Book` property. Make sure this is not duplicated.